backend/endpoints/rooms.py

```diff
@@ -439,6 +439,8 @@ async def get_members(org_id: str, room_id: str):
         ),
     )
 
+<<<<<<< HEAD
+=======
```

Comment:
please remove these merge conflicts markers

Reply:
> please remove these merge conflicts markers

have resolved the conflict sir